translations/pt/get5.phrases.txt

```diff
@@ -0,0 +1,320 @@
+"Phrases"
+{
+    "ReadyToVetoInfoMessage"
+    {
+        "pt"            "Digite {GREEN}!ready {NORMAL}quando seu time estiver pronto para o veto."
+    }
+    "WaitingForCastersReadyInfoMessage"
+    {
+        "pt"            "Esperando pelos streammers para digitar {GREEN}!ready {NORMAL}pra começar."
+    }
+    "ReadyToRestoreBackupInfoMessage"
+    {
+        "pt"            "Digite {GREEN}!ready {NORMAL}quando vocês estiver pronto para restaurar o backup da partida."
+    }
+    "ReadyToKnifeInfoMessage"
+    {
+        "pt"            "Digite {GREEN}!ready {NORMAL}quando você estiver pronto pro Round Faca."
+    }
+    "ReadyToStartInfoMessage"
+    {
+        "pt"            "Digite {GREEN}!ready {NORMAL}quando você estiver pronto para começar."
+    }
+    "YouAreReady"
+    {
+        "pt"            "Você foi marcado como {GREEN}pronto{NORMAL}."
+    }
+    "YouAreNotReady"
+    {
+        "pt"            "Você foi desmarcado como {GREEN}pronto{NORMAL}."
+    }
+    "WaitingForEnemySwapInfoMessage"
+    {
+        "#format"       "{1:s}"
```

Comment:
Remove these #format lines in the translation file. 

See https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#Distributing_Language_Files